CMakeLists.txt

```diff
@@ -56,6 +56,8 @@ if(USE_SANITIZERS)
     else()
         add_compile_options("-fsanitize=address,undefined")
         add_link_options("-fsanitize=address,undefined")
+        add_compile_options("-fno-sanitize-recover=all")
```

Comment:
Will this then crash on any leaks as well?

Reply:
It will, but my intent is to only have this on with the unit tests currently.

Reply:
So now, these options are under USE_FATAL_SANITIZERS